### PR TITLE
Add career plan && fix filter bugs in target && fix bugs of active li…

### PR DESCRIPTION
**Pull request by MehdiElrafi**
…nks (sidebar\)

### CODE REVIEW

**Comment by MehdiElrafi** [comment]
[Développer la partie « plan de développement » dans le module performance](https://trello.com/c/X7Vc5jUX/326-d%C3%A9velopper-la-partie-plan-de-d%C3%A9veloppement-dans-le-module-performance)

File: app/controllers/target_sets_controller.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def index
     authorize!
     scope = Scopes::TargetSetScope.new(user: current_user, type: session[:type])
     @q = scope.relation_scope(TargetSet.where(target_campaign: @target_campaign, organization: @current_organization)).ransack(params[:q])
-    @target_sets = @q.result.order("users.last_name ASC, users.first_name ASC").page(params[:page])
```
Comment (Mouados-byte) [inline app/controllers/target_sets_controller.rb:9]: We had a task about sorting the target sets by name

File: app/helpers/career_plan_sets_helper.rb [hunk 2]
```diff
@@ -0,0 +1,20 @@
+module CareerPlanSetsHelper
+  def career_plan_set_state(career_plan_set)
+    filtered_attributes = career_plan_set.career_plan&.attributes&.reject { |key, _| key.to_s.end_with?("_id", "_at", "id") }
```
Comment (Mouados-byte) [inline app/helpers/career_plan_sets_helper.rb:3]: shouldnt this logic be in a model method that return the status of the career plan set ? , so you can use it everywhere you want other than just the views

Comment (itsAndL) [reply app/helpers/career_plan_sets_helper.rb:3]: we just need it for the view
Aslo remember Skinny Model

**Comment by itsAndL** [comment]
@MehdiElrafi i think you need to create for the last year not the current year
'cause we will evaluate him on the stuff that he does the last year

**Comment by MehdiElrafi** [comment]
@itsAndL Yesterday I think a lot about this point but I will ask, Thank you

**Comment by itsAndL** [comment]
@MehdiElrafi in this part i test with a big text
image

when i did save i get this
image

**Comment by itsAndL** [comment]
@MehdiElrafi i think it well be better to rename this
image

**Comment by itsAndL** [comment]
also @MehdiElrafi this list not sorted

image

**Comment by itsAndL** [comment]
@MehdiElrafi also the filter i can get the list of job titles associated with a job family

image

**Comment by itsAndL** [comment]
also in objectives the same
but in competences, i get all job titles, not the ones that are specific to Job family

**Comment by itsAndL** [comment]
also after i do a filter the breadcumbs not updated
both career plan and objectives
image

**Review by itsAndL — APPROVED** [review]

**Pull request merged**